packages/fdc3-context/test/validate-schema-examples.test.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright FINOS FDC3 contributors - see NOTICE file
+ */
+
+// This test validates that all examples in each JSON schema in schemas/context are valid according to their schema.
+import fs from 'fs';
+import path from 'path';
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+
```

Comment:
Filename sanitizing fn needed below to past SAST scans:

```suggestion
const sanitizeFilename = (filename: string) => filename.replace(/^(\.\.(\/|\\|$))+/, '');
```